src/Microsoft.Framework.Runtime/NuGet/Authoring/PackageDependencySet.cs

```diff
@@ -11,36 +11,46 @@ namespace NuGet
 {
     public class PackageDependencySet : IFrameworkTargetable
     {
-        private readonly FrameworkName _targetFramework;
-        private readonly ReadOnlyCollection<PackageDependency> _dependencies;
-
-        public PackageDependencySet(FrameworkName targetFramework, IEnumerable<PackageDependency> dependencies)
+        public PackageDependencySet(IEnumerable<PackageDependency> dependencies)
```

Comment:
Call the other ctor:

```
public PackageDependencySet(IEnumerable<PackageDependency> dependencies)
 : this(null, dependencies) 
{}
```
(You may need to cast null to select the right overload)